### ISSUE TITLE
Datapackage code

### ISSUE THREAD

**Issue by uvchik** [oemof/oemof#554]
In the EnergySystem class i found the following code block, which I do not understand. All tests pass if I remove this block.

Is this block still needed?

A time ago we decided not to use relative imports but as I do not understand the code it might be necessary in this case.

```python
try:
    from .tools.datapackage import deserialize_energy_system
    from_datapackage = classmethod(deserialize_energy_system)
except ImportError as e:
    @classmethod
    def from_datapackage(cls, *args, **kwargs):
        raise e
```

@oemof/oemof-solph May anybody help?

**Comment by gnn** [comment]
Yeah, that code can be removed. I actually already did so on [`features/move-datapackage-reader-into-standalone-package`](/oemof/oemof/tree/features/move-datapackage-reader-into-standalone-package). If I open a PR for this, should I request a pull into [`dev`](/oemof/oemof) or [`releases/v0_3_0`](/oemof/oemof/tree/releases/v0_3_0)?
Sidenote: you can use dots in branch names. ;)

Concerning relative imports: no, they are not necessary here. I just didn't remember that we decided not to use them. But thanks for the reminder.

**Comment by uvchik** [comment]
> If I open a PR for this, should I request a pull into [`dev`](/oemof/oemof) or [`releases/v0_3_0`](/oemof/oemof/tree/releases/v0_3_0)?
> Sidenote: you can use dots in branch names. ;)

Thanks for the hint :+1: 

If there changes on the dev I merge them into v0_3_0, so if the API is not affected it is your choice. If the API is changed you have to use the release branch.

**Comment by uvchik** [comment]
Fixed by 9b167af .
